maintainers/scripts/update.py

```diff
@@ -228,24 +230,23 @@ async def run_update_script(
             f"UPDATE_NIX_ATTR_PATH={package['attrPath']}",
             *update_script_command,
             stdout=asyncio.subprocess.PIPE,
-            stderr=asyncio.subprocess.PIPE,
+            stderr=asyncio.subprocess.STDOUT,
```

Comment:
If we end up writing everything into the logfile anyway, can we not just pass the file descriptor to the log file instead of using a pipe that has a limited log buffer size and could end up blocking the execution?

Reply:
Currently the log file is only created after a failure, and stdout/err are both buffered in memory.

Reply:
Actually deadlocks are not an issue, when I am thinking about it. Python will read from process pipes to on its own.